_posts/august-2024-newsletter.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "August Newsletter"
+excerpt: "Our August newsletter highlights key events, including our General Meeting on August 18 and upcoming public meetings at the DART Board and Dallas City Council."
+coverImage: "/assets/blog/august-2024-newsletter/cover.jpg"
+date: "2024-08-08T05:00:00.000Z"
+author:
+  name: DATA Newsletter
+  picture: "/assets/blog/authors/data-team.png"
+ogImage:
+  url: "/assets/blog/august-2024-newsletter/cover.jpg"
+---
+Hello there,  
+
+We have three upcoming events that I want you to be aware of this month.  
```

Comment:
Is this intended to be written like it's a letter from someone?

Reply:
Originally yes, this was pulled directly from the email we are sending out. I'm just now realizing it should be slightly more formal

Reply:
It’s okay if it’s like this but I think it would need to be consistent all the way though